src/components/StudentSelect.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <p>SELECT YOUR SCHOOL:</p>
+  <select v-model="selectedSchool" @change="changeSchool(event)">
+    <option disabled value="">Please select your school</option>
+    <option v-for="school in this.schoolsList" :key="school.id" :value="school.name">
+      {{ school.name }}
+    </option>
+  </select>
+
+  <div v-if="this.selectedSchool != null">
```

Comment:
```suggestion
  <div v-if="selectedSchool != null">
```
This should give an error ideally, not sure why it's not giving an error.
Maybe there's an error in console

Reply:
Also, make this a computed variable like `isSelectedSchoolNull`

Reply:
why do you recommend this? Does it make the code cleaner? or is there some other rationale?